spikeinterface/toolkit/postprocessing/principal_component.py

```diff
@@ -337,75 +337,75 @@ def _run_concatenated(self, component_memmap):
 
 def _all_pc_extractor_chunk(segment_index, start_frame, end_frame, worker_ctx):
     recording = worker_ctx['recording']
-    all_pcs = worker_ctx['all_pcs']
+    all_pcs = np.lib.format.open_memmap(**worker_ctx['all_pcs_args'])
```

Comment:
NO NO NO!
Here you open a file every chunk this is speed killer!
This must be done in _init_work_all_pc_extractor